plugins/modules/aci_interface_policy_leaf_fc_policy_group.py

```diff
@@ -0,0 +1,361 @@
+description:
+- Manage fabric interface policy fiber policy groups on Cisco ACI fabrics.
+options:
+  policy_group:
+    description:
+    - Name of the fiber policy group to be added/deleted.
+    type: str
+    aliases: [ name, policy_group_name ]
+  description:
+    description:
+    - Description for the fiber policy group to be created.
+    type: str
+    aliases: [ descr ]
+  lag_type:
+    description:
+    - Selector for the type of fiber policy group we want to create.
+    - C(port) for Fiber Channel (FC)
+    - C(port_channel) for Fiber Channel Port Channel (FC PC)
+    type: str
+    required: true
+    choices: [ port, port_channel ]
+    aliases: [ lag_type_name ]
+  fibre_channel_interface_policy:
+    description:
+    - Choice of fibre_channel_interface_policy to be used as part of the fiber policy group to be created.
+    type: str
+    aliases: [ fibre_channel_interface_policy_name ]
+  port_channel_policy:
+    description:
+    - Choice of port_channel_policy to be used as part of the fiber policy group to be created.
+    type: str
+    aliases: [ port_channel_policy_name ]
+  attached_entity_profile:
+    description:
+    - Choice of attached_entity_profile (attached_entity_profile) to be used as part of the fiber policy group to be created.
+    type: str
+    aliases: [ aep_name, aep ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- When using the module please select the appropriate link_aggregation_type (lag_type).
+- C(port) for Fiber Channel(FC), C(port_channel) for Fiber Channel Port Channel(VPC).
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC classes B(infra:FcAccPortGrp) and B(infra:FcAccBndlGrp).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Anvitha Jain (@anvjain)
+"""
+
+EXAMPLES = r"""
+- name: Create a Fiber Channel (FC) Interface Policy Group
+  cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    lag_type: port
+    fibre_channel_interface_policy: whateverfcinterfacepolicy
+    description: policygroupname description
+    attached_entity_profile: whateveraep
+    state: present
+  delegate_to: localhost
+
+- name: Create a Fiber Channel Port Channel (FC PC) Interface Policy Group
+  cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    lag_type: port_channel
+    fibre_channel_interface_policy: whateverfcinterfacepolicy
```

Comment:
It would be good to add values/name to the attributes below if it is not a hassle ->
Example-
attached_entity_profile: whateveraep -> attached_entity_profile: aep

Reply:
changed.